crypto/rand/drbg_ctr.c

```diff
@@ -51,7 +52,7 @@ static void ctr_XOR(RAND_DRBG_CTR *ctr, const unsigned char *in, size_t inlen)
     for (i = 0; i < n; i++)
         ctr->K[i] ^= in[i];
     if (inlen <= ctr->keylen)
-        return;
+        return 0;
```

Comment:
What makes you think that this is an error?

Reply:
No idea why that's there, fixed.

Reply:
done